AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -161,6 +161,54 @@ private var runtimeTargets: [Target] {
             ],
```

Comment:
Adds two new targets: `InternalAWSSSO` and `InternalAWSSSOOIDC`. Just like `InternalAWSSTS`, they are service clients generated for internal use with only the couple operations needed for the SSO credential resolver.